responders/src/API/EMBC.Responders.API/Controllers/ProfileController.cs

```diff
@@ -185,8 +186,7 @@ public class SecurityMapping : Profile
         public SecurityMapping()
         {
             CreateMap<EMBC.ESS.Shared.Contracts.Team.TeamMember, UserProfile>()
```

Comment:
Would it make more sense to map teammember to teammember and not user userprofile for updates?